templates/applications/backend-app/manifest.yml

```diff
@@ -0,0 +1,53 @@
+# The manifest for the "{{.Name}}" application.
+# Read the full specification for the "{{.Type}}" type at:
+#  https://github.com/aws/amazon-ecs-cli-v2/wiki/Manifests#backend-app
+
+# Your application name will be used in naming your resources like log groups, services, etc.
+name: {{.Name}}
+
+# Your application is reachable at "http://{{.Name}}.${ECS_APP_DISCOVERY_ENDPOINT}:{{.Image.Port}}" but is not public.
+type: {{.Type}}
+
+image:
+  # Path to your application's Dockerfile.
+  build: {{.Image.Build}}
+  # Port exposed through your container to route traffic to it.
+  port: {{.Image.Port}}
+
+# Optional. Container healthcheck settings.
+# See https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html
+#{{if not .HealthCheck}}
+#healthcheck:
+#  #  The command the container runs to determine if it's healthy.
```

Comment:
@kohidave What do you think of moving this inside the `image` field above since it's a value extracted from the Dockerfile?

Reply:
Yea, I think that makes sense. I think for side cars it might be confusing to have one generic "healthcheck" section.